cmd/k8s-bigip-ctlr/main.go

```diff
@@ -419,7 +446,8 @@ func main() {
 		log.Fatalf("error connecting to the client: %v", err)
 	}
 	if *manageRoutes {
-		appMgrParms.RouteClientV1, err = routeclient.New(config)
+		rclient, err := routeclient.New(config)
```

Comment:
Perhaps. I guess leave manageRoutes as-is for now until we work through all the details, etc.